build/lib/etest/__init__.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2014 by Alex Brandt <alunduil@alunduil.com>
```

Comment:
Hmm, build shouldn't be included.  I'll update the gitignore in another pull request so we can rebase and ensure these artifacts aren't included.

Reply:
Can #68 be rebased from github or do we need to do it from git cli?

Reply:
I don't know if the UI can be used for this.  I always do it through the CLI.  Let me know if you run into issues getting it rebased and don't forget to uncommit the build directory before the rebase to make the rebase simpler.

Reply:
I usually do it from the CLI as well, but I wasnt home so I didnt have access to my local repo.
Ill rebase as soon as I get home.